pages-sections/MapSection.js

```diff
@@ -19,77 +19,112 @@ const VectorMap = dynamic(
-        focusOn={{
-          x: 0.6,
-          y: 0.6,
-          scale: 2,
-          animate: true
-        }}
-        panOnDrag={true}
-        showTooltip={true}
-        regionStyle={{
-          initial: {
-            fill: "#e4e4e4",
-            "fill-opacity": 0.9,
-            //stroke: "none",
-            "stroke-width": 0,
-            "stroke-opacity": 0
-          }
-          // hover: {
-          //   "fill-opacity": 0.8,
-          //   cursor: "pointer"
-          // }
-          // selected: {
-          //   fill: "#2938bc" //color for the clicked country
-          // }
-          // selectedHover: {}
-        }}
-        //onRegionTipShow={function(e, el, code){
-        //  el.html("fdsjfds");
-        //}}
-        // regionsSelectable={true}
-        onRegionTipShow={toolTipCountry}
-        //onRegionOver={toolTipCountry}
-        //onLabelShow={toolTipCountry}
-        series={{
-          regions: [
-            {
-              values: mapData, //this is your data
-              scale: ["#146804", "#ff0000"], //your color game's here
-              normalizeFunction: "polynomial"
-            }
-          ]
-        }}
-      />
-    </div>
+      <div className={classes.section}>
+        <GridContainer justify="center">
+          <GridItem cs={12} sm={12} md={12}>
+            <h2 className={classes.title}>Countries Surveyed</h2>
+            <h4 className={classes.description}>
+              Highest number of responses from X <br/>
+              X Responses from Africa, etc.
+            </h4>
+              <GridContainer justify="center">
+                <GridItem xs={12} sm={12} md={12}>
+                  <VectorMap
+                    map={"world_mill"}
+                    backgroundColor="transparent" //change it to ocean blue: #0077be
+                    zoomOnScroll={false}
+                    containerStyle={{
+                      width: "100%",
+                      height: "700px"
+                    }}
+                    onRegionClick={handleClick} //gets the country code
+                    containerClassName="map"
+                    focusOn={{
+                      x: 0.6,
+                      y: 0.6,
+                      scale: 2,
+                      animate: true
+                    }}
+                    panOnDrag={true}
+                    showTooltip={true}
+                    regionStyle={{
+                      initial: {
+                        fill: "#e4e4e4",
+                        "fill-opacity": 0.9,
+                        //stroke: "none",
+                        "stroke-width": 0,
+                        "stroke-opacity": 0
+                      }
+                      // hover: {
+                      //    cursor: "auto"
+                      // }
+                      // selected: {
+                      //   fill: "#2938bc" //color for the clicked country
+                      // }
+                      // selectedHover: {}
```

Comment:
Should we just get rid of this?

Reply:
I will delete this soon! I want to switch the cursor and maybe add a selected field, but since we're showing them tomorrow I didn't have time to finish but I'd like to keep the template until I get to it